__tests__/app.test.js

```diff
@@ -176,6 +176,49 @@ describe('/api/articles', () => {
     })
 })
 
+describe('/api/articles/:article_id/comments', () => {
+    test('GET 200: responds with an array of comments for given article_id with correct properties', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then(({body: {comments}}) => {
+            expect(comments.length).toBe(2)
+            comments.forEach((comment) => {
+                expect(typeof comment.comment_id).toBe('number')
+                expect(typeof comment.votes).toBe('number')
+                expect(typeof comment.created_at).toBe('string')
+                expect(typeof comment.author).toBe('string')
+                expect(typeof comment.body).toBe('string')
+                expect(typeof comment.article_id).toBe('number')
+            }) 
+        })
+    })
+    test('GET 200: responds with comments sorted by date descending', () => {
```

Comment:
we could also have a test for if the article id is valid but there are no associated comments for that article